styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -320,6 +320,8 @@ public void create(Environment environment) {
     final Config staleStateTtlConfig = config.getConfig(STYX_STALE_STATE_TTL_CONFIG);
     final TimeoutConfig timeoutConfig = TimeoutConfig.createFromConfig(staleStateTtlConfig);
 
+    migrateWorkflows(storage);
```

Comment:
Would prefer to run workflow migration as a separate application instead of blocking scheduler startup.